### PR TITLE
Strip quotes from nuget.exe path

### PR DESCRIPTION
**Pull request by jmyersmsft**

### CODE REVIEW

File: Tasks/Common/nuget-task-common/NuGetToolRunner.ts [hunk 1]
```diff
@@ -139,7 +141,11 @@ function locateTool(tool: string, opts?: LocateOptions) {
 
 export function locateNuGetExe(userNuGetExePath: string): string {
     if (userNuGetExePath) {
-        tl.debug(`using user-supplied NuGet path ${userNuGetExePath}`)
+        if (os.platform() === "win32") {
```
Comment (bryanmacfarlane) [inline Tasks/Common/nuget-task-common/NuGetToolRunner.ts]: I think this is likely fine in your case.  We discussed for lib and the reason we didn't do it there is double quote is valid in OSX and Linux in a dir name.  So we can't blindly do it in a base lib.

Comment (jmyersmsft) [inline Tasks/Common/nuget-task-common/NuGetToolRunner.ts]: yep, this is literally just to resolve that one break and even then only on windows.

**Comment by jmyersmsft** [comment]
@prashkan90 Can you take a look at this since you were asking about the bug before?

**Comment by prashkan90** [comment]
Looks good to me. :shipit:

**Pull request merged**